packages/zowe-explorer/__tests__/__unit__/command/tsoCommandHandler.extended.unit.test.ts

```diff
@@ -10,53 +10,73 @@
-            mergeArgsForProfile: jest.fn().mockReturnValue({
-                knownArgs: [{ argName: "account", argValue: "TEST" }],
-            }),
-        }),
+    Object.defineProperty(Profiles, "getInstance", { value: jest.fn(), configurable: true });
+
+    Object.defineProperty(Profiles, "getInstance", {
+        value: jest.fn().mockReturnValue(newMocks.profileInstance),
         configurable: true,
     });
 
     describe("getTsoParms", () => {
         it("should work with teamConfig", async () => {
             Object.defineProperty(Profiles, "getInstance", {
-                value: jest.fn(() => ({ getCliProfileManager: () => null })),
+                value: jest.fn(() => ({
+                    getCliProfileManager: () => null,
+                    getProfileInfo: jest.fn().mockReturnValue({
+                        usingTeamConfig: true,
+                        getAllProfiles: jest.fn().mockReturnValue(["dummy"]),
+                        mergeArgsForProfile: jest.fn().mockReturnValue({
+                            knownArgs: [{ argName: "account", argValue: "TEST" }],
+                        }),
+                    } as any),
+                })),
             });
             const result = await (TsoCommandHandler.getInstance() as any).getTsoParams();
             expect(result.account).toEqual("TEST");
         });
 
         it("should work with teamConfig and prompt if account is empty", async () => {
             Object.defineProperty(Profiles, "getInstance", {
-                value: jest.fn(() => ({ getCliProfileManager: () => null })),
+                value: jest.fn(() => {
+                    return {
+                        getCliProfileManager: () => null,
+                        getProfileInfo: jest.fn().mockReturnValue({
+                            usingTeamConfig: true,
+                            getAllProfiles: jest.fn().mockReturnValue(["dummy"]),
+                            mergeArgsForProfile: jest.fn().mockReturnValue({
+                                knownArgs: [{ argName: "account", argValue: "" }],
+                            }),
+                        } as any),
+                    };
+                }),
             });
 
-            jest.spyOn(globals.PROFILESCACHE, "getProfileInfo").mockReturnValue({
-                usingTeamConfig: true,
-                getAllProfiles: jest.fn().mockReturnValue(["dummy"]),
-                mergeArgsForProfile: jest.fn().mockReturnValue({
-                    knownArgs: [{ argName: "account", argValue: "" }],
-                }),
-            } as any);
+            // jest.spyOn(Profiles.getInstance, "getProfileInfo").mockReturnValue({
+            //     usingTeamConfig: true,
+            //     getAllProfiles: jest.fn().mockReturnValue(["dummy"]),
+            //     mergeArgsForProfile: jest.fn().mockReturnValue({
+            //         knownArgs: [{ argName: "account", argValue: "" }],
+            //     }),
+            // } as any);
```

Comment:
Seems like this should be good to remove now that it's in the above  `defineProperty` call 😋

Reply:
good catch, pushed cleanup